pagefaultgames/pokerogue
# [Localization] Rival dialogue EN localisation fix

## What are the changes the user will see?
The male rival's English dialogue has two instances where a new line didn't copy over from #3705. Because of this, there's no space between the punctuation and the next word. This PR fixes it

## Why am I making these changes?
The male rival's English dialogue has two instances where a new line didn't copy over from #3705. Because of this, there's no space between the punctuation and the next word. This PR fixes it

## What are the changes from a developer perspective?
The male rival's English dialogue has two instances where a new line didn't copy over from #3705. Because of this, there's no space between the punctuation and the next word. This PR fixes it

### Screenshots/Videos
Current beta: 

![image](https://github.com/user-attachments/assets/2050ad97-53a2-4bb9-91a9-83afe5f19d6f)

![image](https://github.com/user-attachments/assets/e9e59394-5fdf-4d20-a0fe-a4c1732ca504)

After fix:

![image](https://github.com/user-attachments/assets/164d6f60-a712-4d5d-9015-de61209608cd)

![image](https://github.com/user-attachments/assets/ed49a832-f3e4-435a-be61-b494d75333e8)

This is an example of what it's like in live:

![image](https://github.com/user-attachments/assets/e2574007-a0f0-4486-9c9e-d3365910a2a6)

![image](https://github.com/user-attachments/assets/54c5620e-332f-4598-8d8b-dee588b4d29b)

## How to test the changes?
Download this PR, start a new game (make sure your gender is female), get to wave 8 and see the male rival's dialogue has changed.

## Checklist
